src/main/java/io/confluent/kafkarest/KafkaRestMain.java

```diff
@@ -50,8 +52,22 @@ public boolean verify(
    */
   public static void main(String[] args) throws IOException {
     try {
+      /*TODO discuss with team and look at options to do something like Apache CLI here. Handle
+        exception for incorrect Class args with appropriate message*/
       KafkaRestConfig config = new KafkaRestConfig((args.length > 0 ? args[0] : null));
-      KafkaRestApplication app = new KafkaRestApplication(config);
+
+      RestResourceExtension restResourceExtension = null;
+      String extensionClassName = config.getString(KafkaRestConfig
+                                                       .KAFKA_REST_RESOURCE_EXTENSION_CONFIG);
+      if (StringUtil.isNotBlank(extensionClassName)) {
+        Class<RestResourceExtension>
```

Comment:
Presumably this will fail if the configured class doesn't implement `RestResourceExtension`. Should we fail with a more descriptive error?